vrk-kpa/suomidigi
# SUOPA-597 Community content type

To test: 
- `make fresh`
  - **It will fail to import configuration so after the failure, run** `make drush-cr drush-cim drush-uli`
   - The plain old make fresh is not enough, because of the caches, entity removal/creation and configuration import order
- Log in, go to https://suomidigi.docker.sh/node/add/landing_page and create a test landing page with a title and **view** paragraph. From paragraph settings, select **Communities** and **Block**. Save the node.
- Open a new browser window in incognito and head to https://suomidigi.docker.sh/form/add-new-community
   - Create a new community via this form
- As a logged in user, head to https://suomidigi.docker.sh/admin/content/approval-queue and publish and approve the community what you've just created through the webform.
- Head to the node what you've created earlier and refresh it. You should now see the community in the page.
